lua/octo/utils.lua

```diff
@@ -71,6 +71,30 @@ M.file_status_map = {
   renamed = "R",
 }
 
+M.checks_hl_map = {
+  ERROR = "OctoStateDismissed",
+  EXPECTED = "OctoStatePending",
+  FAILURE = "OctoStateDismissed",
+  PENDING = "OctoStatePending",
+  SUCCESS = "OctoStateApproved",
+}
+
+M.checks_icon_map = {
+  ERROR = "× ",
+  EXPECTED = " ",
+  FAILURE = "× ",
+  PENDING = " ",
+  SUCCESS = "✓ ",
+}
+
+M.checks_message_map = {
+  ERROR = "errored",
+  EXPECTED = "expected",
+  FAILURE = "failed",
+  PENDING = "pending",
+  SUCCESS = "passed",
+}
```

Comment:
These will always go together, right? Maybe combine in table before any lookup

for instance, the first entry is this
{ "x errored", "OctoStateDismissed" }